Issue 125

jdbrown-USGS opened the pull request:
Fixes usgs/hazdev-template#125

- Updated alert styles to be more like 18F project
- Added alert icons and a style to remove alert icons.

ehunter-usgs commented:
We talked about:
 - Add more padding/space to the alert icons (top/left: 1em)
 - Combine redundant styles in alert.scss
 - Adhere to css coding styles by adding new lines before nested styles
 - Add multi-line examples to alert.php, maybe one with an h3?

ehunter-usgs commented:
Pertaining to working copies of svg images,
 - we only need to save alerts.eps
 - alerts.eps should not be included in the build (create a separate directory for working copies of image files)

ehunter-usgs commented:
Update fill on alerts.svg icons to be #444444 instead of true black

jdbrown-USGS commented:
- Added padding/Space to alert icon.
- Combined redundant styles.
- Fixed css coding styles.
- Added multi line example to alert with header.
- Removed un needed eps files.
- Excluded .eps files in copy.
- Changed fill color on svg.

ehunter-usgs commented:
I missed some things while reviewing this.
 - Since this has already been merged, make sure you rebase.
 - There should only be one usage block for code preview
 - Update the usage example so that they match the new markup
 - .alert { margin: 1em 0 0;}
